aries_cloudagent/connections/base_manager.py

```diff
@@ -254,6 +254,7 @@ async def resolve_invitation(
         try:
             doc_dict: dict = await resolver.resolve(self._profile, did, service_accept)
             doc: ResolvedDocument = pydid.deserialize_document(doc_dict, strict=True)
+            self._logger.warning("Resolved doc ===> ", doc)
```

Comment:
`warning` is a curios log level to choose for this; I think I'd recommend info or debug. Is there a reason you selected `warning` specifically?

Reply:
Hey I am doing some debugging on a forked repo not sure why this ended up here. I am gonna close this.

Reply:
Sorry about wasting your time.